sys-block/vblade/files/vblade-24-fno-common.patch

```diff
@@ -0,0 +1,71 @@
+Taken from: https://github.com/OpenAoE/vblade/pull/15
+Author: Andreas Kölbl <koelblandreas@freenet.de>
+
+--- /dev/null
++++ b/dat.c
+@@ -0,0 +1,12 @@
++/* dat.c: Global memory for vblade AoE target */
++#include "dat.h"
++
++int	shelf, slot;
++ulong	aoetag;
++uchar	mac[6];
++int	bfd;		// block file descriptor
++int	sfd;		// socket file descriptor
++vlong	size;		// size of vblade
++vlong	offset;
++char	*progname;
++char	serial[Nserial+1];
+--- a/dat.h
++++ b/dat.h
+@@ -1,4 +1,5 @@
+ /* dat.h: include file for vblade AoE target */
++#include <sys/types.h>
+ 
+ #define	nil	((void *)0)
+ /*
+@@ -163,12 +164,12 @@ enum {
+ 	Nserial= 20,
+ };
+ 
+-int	shelf, slot;
+-ulong	aoetag;
+-uchar	mac[6];
+-int	bfd;		// block file descriptor
+-int	sfd;		// socket file descriptor
+-vlong	size;		// size of vblade
+-vlong	offset;
+-char	*progname;
+-char	serial[Nserial+1];
++extern int	shelf, slot;
++extern ulong	aoetag;
++extern uchar	mac[6];
++extern int	bfd;		// block file descriptor
++extern int	sfd;		// socket file descriptor
++extern vlong	size;		// size of vblade
++extern vlong	offset;
++extern char	*progname;
++extern char	serial[Nserial+1];
+--- a/makefile
++++ b/makefile
+@@ -8,8 +8,8 @@ sbindir = ${prefix}/sbin
+ sharedir = ${prefix}/share
+ mandir = ${sharedir}/man
+ 
+-O=aoe.o bpf.o ${PLATFORM}.o ata.o
+-CFLAGS += -Wall -g -O2
++O=aoe.o bpf.o ${PLATFORM}.o ata.o dat.o
++CFLAGS += -Wall -g -O2 -fno-common
```

Comment:
this seems wrong, change it to `CFLAGS += -Wall`

Reply:
:+1: Done